### ISSUE TITLE
[Request] Unlimited quantity option for merchants

### ISSUE THREAD

**Issue by iconoplast** [jopeek/fvtt-loot-sheet-npc-5e#112]
Is there a way to have infinite # of items in stock for a merchant in LootSheet?  If not, and if it's not too much trouble, a checkbox marked "do not run out of inventory" or "automatically replenish" or something would be super helpful

**Comment by Streyder** [comment]
You could also just set the quantity of an item to 9999999999999 which should be good enough, I guess :D

**Comment by iconoplast** [comment]
Yeah but setting each item individually to 1,000 or even to 20 is a little time consuming.  Was hoping to see a 'never run out of inventory' that could be set at the merchant level instead of at the item level

**Comment by Streyder** [comment]
Well a on-merchant level would be easy to do, but also limited in functionality. I dont really see the point. Just set the per-item amount during generation of the merchant to 999999 and be done with it? Even setting 20 items shouldnt take too long and this isnt something you do every session but once per campaign usually
